[C#] linked to config.json as the place to search exercises (#1661)
exercism/v3: languages/csharp/exercises/concept/README.md

## Implemented exercises

> If you need to see a list of the completed exercises the follow this [link][config-json] to the C# track's _config.json_ file.  In addtion, in the file, you can search for _concepts_ to ascertain which exercises they are featured in and the _prerequisites_  sections will show the relationships between the exercises.

Reviewer: ```suggestion
The list of completed exercises can be found in the C# track's [`config.json` file][config-json]. In that file, you can search for _concepts_ to ascertain which exercises they are featured in and the _prerequisites_  sections will show the relationships between the exercises.
```

> [config.json][config-json] should be updated, as part of the PR, as the last stage in implementing an exercise.  Note that you will need to generate and include an UUID to identify the exercise.  This also applies when improving an exercise if you modify the set of concepts or prerequisites.

Reviewer: Maybe instead of listing the UUID requirement, we could link to the https://v3.exercism.io/csharp/contributing link to get people on the right track to get started? They don't have to worry about the UUID until the end of the PR process.

```suggestion
The [`config.json` file][config-json] should be updated, as part of the PR, as the last stage in implementing an exercise. This also applies when improving an exercise if you modify the set of concepts or prerequisites.
```
